utils/token_tracker.py

```
@@ -192,6 +192,32 @@ def _file_lock(lock_path: Path, timeout: float = 10.0):
+            if not fname.endswith(".md"):
+                continue
+            stem = fname[:-3]
+            try:
+                ver = tuple(int(x) for x in stem.split("."))
+            except (ValueError, AttributeError):
+                continue
+            if ver > best_ver:
+                best_ver = ver
+                best_stem = stem
+        return best_stem
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)